src/plugins/contextMenu/contextMenu.js

```diff
@@ -236,6 +236,41 @@ class ContextMenu extends BasePlugin {
     this.commandExecutor.execute.apply(this.commandExecutor, params);
   }
 
+  changeItems(removeItemFunction, newItems) {
+
+    if (typeof removeItemFunction === 'function') {
+      arrayEach(Object.entries(this.itemsFactory.predefinedItems), (entry) => {
+        let key = entry[0];
+        let item = entry[1];
+        if (removeItemFunction(key, item)) {
```

Comment:
the first param is a function that takes the item's key and the item, which determines if an item should be removed.